sbpy/bib/core.py

```diff
@@ -89,33 +107,39 @@ def to_text():
         output += '{:s}:\n'.format(task)
         try:
             for key, val in ref.items():
-                with warnings.catch_warnings():
-                    warnings.filterwarnings('error')
-                    try:
-                        # request needed fields to avoid lazy loading
-                        paper = list(
-                            ads.SearchQuery(
-                                bibcode=val,
-                                fl=['first_author', 'author', 'year']
-                            ))[0]
-                    except Warning as e:
-                        # if query failed,
-                        output += '  {:s}: {:s}\n'.format(key, val)
-                        continue
-
-                if len(paper.author) > 4:
-                    author = '{:s} et al.'.format(
-                        paper.first_author.split(',')[0])
-                elif len(paper.author) > 1:
-                    author = ','.join([au.split(',')[0] for au in
-                                       paper.author[:-1]])
-                    author += ' & {:s}'.format(paper.author[-1].split(',')[0])
-                else:
-                    # single author
-                    author = paper.first_author.split(',')[0]
-
-                output += '  {:s}: {:s} {:s}, {:s}\n'.format(key, author,
-                                                             paper.year, val)
+                if isinstance(val, str):
+                    val = [val]
+                val = list(val)
+                for i in range(len(val)):
```

Comment:
can you change this from a loop over indices to a loop over the values? Just like in my example in `bib.register`.